src/main/perf/LineFileDocs.java

```diff
@@ -641,6 +685,16 @@ public Document nextDoc(DocState doc) throws IOException {
         }
       }
 
+      if (facetFields.containsKey("RandomLabel")) {
+        int flag = facetFields.get("RandomLabel");
+        if ((flag & 1) != 0) {
+          doc2.add(new FacetField("RandomLabel.taxonomy", randomLabel));
```

Comment:
Hmm it looks like we are randomizing whether the term goes to taxo facets and SSDV facets ... but I think it'd be better if they were consistent (i.e. same `FacetLabel` goes to each)?  This allows us to compare their relative performance.

If we can switch to picking the random word at LFD-build time that should automatically resolve this.